generators/heroku/index.js

```diff
@@ -203,9 +207,68 @@ module.exports = class extends BaseGenerator {
                     },
                 ];
 
-                this.prompt(prompts).then(props => {
+                return this.prompt(prompts).then(props => {
                     this.herokuJavaVersion = props.herokuJavaVersion;
-                    done();
+                });
+            },
+            askForOkta() {
+                if (this.abort) return null;
+                if (this.authenticationType !== 'oauth2') return null;
+                if (this.useOkta) return null;
+                const prompts = [
+                    {
+                        type: 'list',
+                        name: 'useOkta',
+                        message: 'You are using oauth2. Do you want to use Okta as IAM or configure e.g. keycloak yourself?',
+                        choices: [
+                            {
+                                value: true,
+                                name: 'Yes, provision the Okta addon',
+                            },
+                            {
+                                value: false,
+                                name: 'No, I want to configure the IAM manually',
+                            },
+                        ],
+                        default: 1,
```

Comment:
You should use `defaults: true`.

Reply:
You mean `default: false`? According to https://github.com/SBoudrias/Inquirer.js#list---type-list it can be choice index too.

Reply:
I didn’t remembered that.
I am used to use the value.
- If the choices order changes, we are safe.
- don’t need to count. 😄 

Anyway if this works, no problem.